CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.2.0-rc.1 December 2, 2021
+
+## Features ✨ and improvements 🏁
+* Cancellable queryRenderFeatures ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
+
+## Bug fixes 🐞
+* Do not allow to use one associatedFeatureId with multiple view annotations. ([#896](https://github.com/mapbox/mapbox-maps-android/pull/896))
+* Fix an issue where shove gesture was not detected when angle between touch points are not horizontal. ([#875](https://github.com/mapbox/mapbox-maps-android/pull/875))
+* Fix gestures at high camera pitch near horizon line. ([#927](https://github.com/mapbox/mapbox-maps-android/pull/927))
+* Reduce drag sensitivity around and above the horizon. ([#925](https://github.com/mapbox/mapbox-maps-android/pull/925))
```

Comment:
I guess it's not needed as we have #927 
@pengdev could you please advise here?

Reply:
yes, it's the gl-native part fix of #927, maybe we can merge this entry with #927

Reply:
something like 

* Fix gestures at high camera pitch near horizon line. ([#925](https://github.com/mapbox/mapbox-maps-android/pull/925), [#927](https://github.com/mapbox/mapbox-maps-android/pull/927))